client/components/ReviewRating/RatingBreakDown/Reference.js

```diff
@@ -0,0 +1,10 @@
+const characteristic = {
```

Comment:
This isn't a component, so I'm not sure if it makes sense to live here. Is there any reason why it can't live in the component that uses it?

Reply:
That was the mock data early, I can delete it. thanks

Reply:
Sorry, that was the one I create for my form radio. I do need this.

Reply:
If this is specific to that component, can we include it in that file?

Reply:
I agreed to move it into component.